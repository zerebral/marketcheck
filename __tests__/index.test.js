import React from 'react'
import renderer from 'react-test-renderer'

import Index from '../pages/index.js'

describe('Snapshot test passes', () => {
  it('Passes the snapshot', () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
