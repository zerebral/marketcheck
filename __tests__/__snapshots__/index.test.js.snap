// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test passes Passes the snapshot 1`] = `
<p>
  I'm a menu
</p>
`;
